### PR TITLE
Change git user.name and user.email

### PR DESCRIPTION
**Pull request by sinsukehlab**
### Summary

Change git user.name and user.email to display github-actions[bot] as a contributor.

### Changes

.github/workflows/0-welcome.yml
- git user.name: github-actions &rarr; github-actions[bot]
- git user.email: github-actions@github.com &rarr; github-actions[bot]@users.noreply.github.com

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by dhananjayaaps — APPROVED** [review]

**Review by dhananjayaaps — APPROVED** [review]

File: .github/workflows/0-welcome.yml [hunk 1]
```diff
@@ -74,8 +74,8 @@ jobs:
           sed -i.bak 's/Game over/Game over, refresh to play again 🧑‍💻 🤖!/' index.html
 
           echo "Make a commit"
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name github-actions[bot]
+          git config user.email github-actions[bot]@users.noreply.github.com
```
Comment (JesusAbdiGuillermo) [inline .github/workflows/0-welcome.yml:78]: chionage

**Review by JesusAbdiGuillermo — APPROVED** [review]

**Review by JesusAbdiGuillermo — CHANGES_REQUESTED** [review]
listo

**Review by JesusAbdiGuillermo — APPROVED** [review]

**Review by JesusAbdiGuillermo — APPROVED** [review]

**Comment by JesusAbdiGuillermo** [comment]
asddasda

**Review by tester-1223 — COMMENTED** [review]
ok

**Comment by heiskr** [comment]
Thank you

**Review by Wangtt185 — COMMENTED** [review]
会很快乐

**Pull request merged**